addons/sales_team/tests/test_sales_team_membership.py

```diff
@@ -276,6 +276,27 @@ def test_memberships_sync(self):
             added.write({'crm_team_id': sales_team_1.id})
             self.new_team.flush()
 
+    def test_sales_team_member_search(self):
+        """ when a search is triggered on the member_ids field in crm.team
+        it is currently returning the archived records also. this test will
+        ensure that the search wont return archived record.
+
+        this is to fix unwanted ORM behavior
+        """
+        new_team = self.env['crm.team'].create({'name': 'Test Team'})
+        partner = self.env['res.partner'].create({'name': 'Test Partner'})
+        partner.write({'team_id': new_team.id})
```

Comment:
Is it relevant to the test ?

Reply:
partner.write({'team_id': new_team.id})

regarding this line, let me check and confirm

thanks @tde-banana-odoo

Reply:
updated the test case with minimal need things for the test to work, above line of adding team to the partner is needed and i have moved that inside the create method.